stories/Organism/Text/PostHeader/postheader.scss

```diff
@@ -1,12 +1,10 @@
-@import '../../../assets/scss/variables';
-@import '../../../assets/scss/typography';
-@import '../../../assets/scss/mixins';
+@import "../../../assets/scss/variables";
+@import "../../../assets/scss/typography";
+@import "../../../assets/scss/mixins";
```

Comment:
@Ashish-srij Please lint the code as it should have the single quotes

Reply:
@surbhi90 done.